src/api/test/unit/user_ldap_strategy_test.rb

```diff
@@ -36,7 +36,7 @@ def test_authenticate_with_local
     assert a == true
   end
 
-  def test_dn2user_principal_name
+  def test_dn2user_principal_name # spec/models/user_ldap_strategy_spec.rb
```

Comment:
It was tested in the old test suite but not in the new rspec and we agreed some time ago when we migrate a test we write in a comment the file where it is in the new test suite.